### ISSUE TITLE
Jackson does not serialize @Transient fields in Hibernate

### ISSUE THREAD

**Issue by dblezek** [dropwizard/dropwizard#932]
According to an [issue in Jackson](https://github.com/FasterXML/jackson-databind/issues/573), `@Transient` annotated fields/getters are not serialized to JSON in Jackson > 2.0.  The solution is simple, disable the feature in the Jackson Hibernate4Module, i.e. 

```java
module.disable(Hibernate4Module.Feature.USE_TRANSIENT_ANNOTATION);
```

 However, this method is marked final in `HibernateBundle`, thus there is no way to disable this feature when using the Bundle:

```java
    @Override
    public final void initialize(Bootstrap<?> bootstrap) {
        bootstrap.getObjectMapper().registerModule(new Hibernate4Module());
    }
```

This creates subtle problems when generating JSON from Hibernate objects...

**Comment by mcarrierastonish** [comment]
#663 Should provide a solution for you.

**Comment by dblezek** [comment]
@mcarrierastonish, perfect.  I forgot to mention that we are using Dropwizard `v0.7.1`.  I couldn't immediately tell if the fix is in `0.8` or not.  Closing for now.

**Comment by mcarrierastonish** [comment]
@dblezek The fix is for 0.8. I had to create my own version of the Hibernate4Module to accomplish the same thing for 0.7.1.

**Comment by druidexpo** [comment]
This did not work for me with Dropwizard 1.3.23.

In order to get it working with this version I added following to my pom.xml:

`
<properties>
    <jackson-datatype-hibernate5.version>2.9.10</jackson-datatype-hibernate5.version>
</properties>
...
<dependencies>
        <dependency>
            <groupId>com.fasterxml.jackson.datatype</groupId>
            <artifactId>jackson-datatype-hibernate5</artifactId>
            <version>${jackson-datatype-hibernate5.version}</version>
        </dependency>
<dependencies>
...
`

Dropwizard Application:
`
public class XApplication extends Application<XConfiguration> {
	public static void main(String[] args) throws Exception {
		new XApplication().run(args);
	}

	@Override
	public void run(SuperOfficeConfiguration configuration, Environment environment)throws Exception {
            Hibernate5Module hibernate5Module = new Hibernate5Module();
            hibernate5Module.disable(Hibernate5Module.Feature.USE_TRANSIENT_ANNOTATION);
            environment.getObjectMapper().registerModule(hibernate5Module);
        }

}
`

Hibernate ORM Entity with a JSON serializable transient property:

`
    @JsonProperty
    @Transient
    private String jsonOnlyProp;
`
